src/components/date-range-picker/date-range-picker.component.html

```diff
@@ -2,9 +2,21 @@
 
   <div class="header-section">
     <div *ngIf="!rangeService.start" class="select-header">{{ selectStartTitle }}</div>
-    <div *ngIf="rangeService.start" class="date-header">{{ rangeService.start | date: 'd MMMM y' }}</div>
+      <div *ngIf="rangeService.start" class="date-header">
+          <ng-container *ngIf="!dateLocalization">
```

Comment:
I think we can get rid of these `ngIf`s, instead we could just set the default `dateLocalization` value in the typescript to be:

```
dateLocalization = 'd MMMM y';
```
And then we only need to have:

```html
{{ rangeService.start | dateLocale: dateLocalization }}
```

As it handles dates strings anyway